### DISCUSSION TITLE
OpenNept4une

### DISCUSSION THREAD

**Discussion by Evan1719** [OpenNeptune3D/OpenNept4une#239] [Q&A]
After flashing my Neptune 4 pro with OpenNept4une. I am receiving an MCU Protocol error on my Fluidd dashboard. The MCU's were updated properly as the file was changed to cursor. What is the issue occurring?

**Comment by Grivits** [comment]
I am sure you have already figired this out, but, if not.... Have you updated both the mainboard via the hidden sd card slot and the EMMC using an EMMC adapter? Because typically the issue you are getting is caused by both MCU's not being in sync. The OpenNept4une image has to be flashed using both methods for it to work properly.
